client/src/document/index.tsx

```diff
@@ -150,7 +150,7 @@ export function Document(props /* TODO: define a TS interface for this */) {
 
   return (
     <>
-      {doc.isArchive && !doc.isTranslated && <Archived />}
+      {doc.isArchive && !doc.isTranslated}
```

Comment:
You can now delete this line.

Reply:
Woops! Almost forgot about this, good catch.